webdevbroz/pomodoro-app
# Add timer settings

## Goal
Related issue: https://github.com/webdevbroz/pomodoro-app/issues/23

We want users to have the ability to change the length of their designated `pomodoro`, `short break`, and `long break` timed sessions. This PR adds a module in the settings menu for doing just that.
## How to test
- Change the different time sessions and make sure the time updates and the circle properly ticks down according to the selected time.
- Check the styling against the figma file. Make sure the layouts work for desktop and mobile.

## Screenshots
mobile:
![image](https://github.com/webdevbroz/pomodoro-app/assets/27636896/fcfa7330-30d9-4a25-a0f4-1711b0640ace)

desktop:
![image](https://github.com/webdevbroz/pomodoro-app/assets/27636896/0416ea3b-1e1c-4e37-9510-e61e1f3ac8d0)
